src/smtsolvers/CoreSMTSolver.cc

```diff
@@ -1465,6 +1444,10 @@ lbool CoreSMTSolver::search(int nof_conflicts, int nof_learnts)
 
         CRef confl = propagate();
         if (confl != CRef_Undef) {
+            if (conflicts > conflictsUntilFlip) {
+                conflictsUntilFlip += flipIncrement;
+                flipState = not flipState;
```

Comment:
Ahh cool! I like this fix to have inverse stable polarities too :)

Reply:
This was already in your original PR.  I messed up the updating of `conflictsUntilFlip`, this is brought "back" in commit 45711aad.